portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,15 @@ function showFact() {
     const factContainer = document.getElementById('fact-container');
     factContainer.innerText = fact;
 }
+
+
+async function getMessages() {
+    console.log("Getting the messages...");
+
+    const response = await fetch('data');
+    const messages = await response.json();
+
+    const messageContainer = document.getElementById('message-container');
+    
+    messageContainer.innerText = messages[0] + "\n" + messages[1] + "\n" + messages[2];
```

Comment:
This can be simplified as: messages.join('\n');
This way it also handles when messages has more than 3 messages in it.

Reply:
Didn't know about join(). Thank you!